src/ctest.c

```diff
@@ -239,6 +239,39 @@ size_t RunTests(const TEST_FUNCTION_DATA* testListHead, const char* testSuiteNam
 #endif
 
 #ifdef VLD_OPT_REPORT_TO_STDOUT
```

Comment:
>VLD_OPT_REPORT_TO_STDOUT [](start = 7, length = 24)

Consider opening an issue to have a cmake flag for leak detection.

Reply:
Added task #7989213

---
In reply to: [470361455](https://github.com/Azure/azure-ctest/pull/59#discussion_r470361455) [](ancestors = 470361455)